dtl-lexer/src/variable.rs

```diff
@@ -118,6 +118,9 @@ pub fn lex_variable(
 
     let start = start + variable.len() - rest.len();
     let content = trim_variable(rest);
+    let end = content.len();
+    let at = (start, end);
+
```

Comment:
These can go below the `content.is_empty()` check (currently just below this comment).